src/main/java/uk/gov/hmcts/darts/audio/service/impl/MediaRequestServiceImpl.java

```diff
@@ -74,6 +80,26 @@ public Integer saveAudioRequest(AudioRequestDetails request) {
         return audioRequest.getId();
     }
 
+    @Transactional
+    @Override
+    public void deleteAudioRequest(Integer mediaRequestId) {
+
+        var transientObject = transientObjectDirectoryRepository.getTransientObjectDirectoryEntityByMediaRequest_Id(mediaRequestId);
+
+        if (transientObject.isPresent()) {
+            TransientObjectDirectoryEntity mediaTransientObject  = transientObject.get();
+            UUID blobId = mediaTransientObject.getExternalLocation();
+
+            if (blobId != null) {
+                dataManagementApi.deleteBlobDataFromOutboundContainer(blobId);
+            }
+
+            transientObjectDirectoryRepository.deleteById(mediaTransientObject.getId());
+        }
+
+        mediaRequestRepository.deleteById(mediaRequestId);
```

Comment:
what happens if the requestId which we are trying to delete doesnt exist in Media Request table? Which error the JPA repository thows?